sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/PortSpec.java

```diff
@@ -30,10 +32,12 @@ public PortSpec(
             @JsonProperty("role") String role,
             @JsonProperty("principal") String principal,
             @JsonProperty("env-key") String envKey,
-            @JsonProperty("port-name") String portName) {
+            @JsonProperty("port-name") String portName,
+            @JsonProperty("network-names") Collection<String> networkNames) {
```

Comment:
1. Actually I think we may need to use them soon. See (https://jira.mesosphere.com/browse/INFINITY-1636).
2. The network names vary on a Pod-level, but ports are on a task level. I propagated the `networkNames` to the port spec so that the maximum amount of information can be conveyed to the `getResourceRequirement()` method. From a resource requirement point of view, we need the names so that we know whether to consume port resources or not.